main.go

```diff
@@ -223,6 +223,26 @@ func getBasicInfoOfState(c echo.Context) error {
 	return c.JSON(http.StatusOK, state)
 }
 
+func getMostRecentYearAndMonth(c echo.Context) error {
```

Comment:
Tem certeza que você não pode extrair o que você quer do método `fetchData`? Lá ele já dá todos um resumo dos dados mensais para um determinado ano.

Note que aqui você vai precisar fazer 11 chamadas remotas quando o mês corrente for dezembro (imagine agora uma galera acessando, lascou) hahahah